src/raftstore/store/peer.rs

```diff
@@ -752,6 +752,11 @@ impl Peer {
     }
 
     pub fn check_stale_state(&mut self) -> StaleState {
+        if self.is_leader() {
+            // Leaders always have valid state.
+            self.leader_missing_time = None;
```

Comment:
need test to cover it?

Reply:
It is hard to test, it only prints log when the bug is triggered.

Reply:
em, maybe you can use fail-rs?